README.md

```diff
@@ -19,6 +19,12 @@ The xontrib will get loaded automatically.
 
 ## Usage
 
+Enable in your [Run Control (RC) file](https://xon.sh/xonshrc.html):
```

Comment:
This looks good, but from a documentation point of view I'd like to not recommend this pattern by default.  It's perfectly valid, but let's promote dynamic enabling of secrets and not permanently exposed secrets like this.

Can you qualify this initial statement with something like this:
> For maximum security, enable your secrets dynamically using the command below when required. You may also enable permanently and expose your secrets  at all times in your [Run Control (RC) file](https://xon.sh/xonshrc.html):

Reply:
Updated to your version!